build.sh

```diff
@@ -93,6 +93,8 @@ WB_CHECK=
 WB_ANALYZE=
 WB_ARGS=
 TARGET_PATH=0
+# -DCMAKE_EXPORT_COMPILE_COMMANDS=ON useful for clang-query tool
+CMAKE_EXPORT_COMPILE_COMMANDS="-DCMAKE_EXPORT_COMPILE_COMMANDS=ON"
```

Comment:
>CMAKE_EXPORT_COMPILE_COMMANDS [](start = 0, length = 29)

You can delete this "CMAKE_EXPORT_COMPILE_COMMANDS"... Too little chance anybody will ever use it.